### PR TITLE
Fix CI: "Support" all Python v3 versions

### PR DESCRIPTION
**Pull request by jedie**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = 'bx_py_utils'
-version = '0.0.1.dev0'
+version = "0.0.1.dev0"
```
Comment (phihag) [inline pyproject.toml:3]: why double quotes?

Comment (jedie) [inline pyproject.toml:3]: `poetry version` replaces the Quotation marks :(

Comment (jedie) [reply pyproject.toml:3]: Quotation marks

Comment (jedie) [reply pyproject.toml:3]: poetry version replaces the Quotation marks :(

Comment (jedie) [reply pyproject.toml:3]: reported here: https://github.com/python-poetry/poetry/issues/3226

**Pull request merged**